lib/id3c/cli/redcap.py

```diff
@@ -255,10 +258,15 @@ def is_complete(instrument: str, data: dict) -> bool:
     True
     >>> is_complete("test", {"test_complete": "Incomplete"})
     False
-    >>> is_complete("test", {})
-    False
+    >>> is_complete("test", {}) is None
+    True
```

Comment:
The is_complete("test", {}) case will return None or True? Is the "True" dangling here?

Reply:
`is_complete("test", {})` should return `None`. This is checking `is_complete("test", {}) is None`, which should return `True`. 

There might be a better way of writing this particular test?

Reply:
That makes sense!